rdr_service/api/base_api.py

```diff
@@ -22,7 +22,7 @@
 
 
 DEFAULT_MAX_RESULTS = 100
-MAX_MAX_RESULTS = 10000
+MAX_MAX_RESULTS = 6000
```

Comment:
This update affects the Paricipant Summary API too. Is that intended, or should the code be updated to only set this as the max for the research workbench API's?

Reply:
This was intended, but after discussion in Slack, I will override the value instead to avoid any errors for awardees currently calling the PaarticipantSummary API with `_count > 6000`